vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,107 @@
+# Reconcile Mouse APIs in React Native - Desktop
+
+Windows and macOS support mouse devices today, but their React Native counterparts are not fully flushed out yet to unblock customers attempting to create React Native apps on those devices.
+
+This document will attempt to capture what's needed within React Native for Windows and macOS to enable customer scenarios. This includes:
+
+- Aligning with the [react-native-web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0) mouse behavior/implementation [issue TBD]()
+- Ensuring mouse behavior is consistent across the two platforms we support
+- Once our two desktop platforms are aligned with React Native Web, we upstream this implementation to core [issue TBD]()
+
```

Comment:
Just a general overall question / observation:
* Android has mouse support through USB OTG / BlueTooth.
* Ios (at least IPad) has mouse support.
So my questions are:
* Is react-native for those platforms considering mouse? 
* Do we have the stakeholders on board to ensure we don't introduce blockers?
* Should we make sure the implementation is backportable to react-native repo to have identical behavior on all platforms?

Reply:
This is good to know! I was trying to cover the absolute base minimum needed to enable mouse on windows first, then we can look at others. Either way, I should probably add these notes to be observed for later consideration.